src/profile-logic/mozilla-symbolication-api.js

```diff
@@ -32,6 +32,12 @@ type APIFrameInfoV5 = {
   // The hex offset between the requested address and the start of the function,
   // e.g. "0x3c".
   function_offset?: string,
+  // The path of the file that contains the function this frame was in, optional.
```

Comment:
nit: I think it would be good to explain why they are optional here (and other places) as well. With something like: `Optional because older implementation doesn't have this ability` Assuming that it will be non-optional when we switch to this implementation. What do you think?

Reply:
I've added a comment.